main.js

```diff
@@ -92,24 +92,36 @@ function getMainMapAddrs(json) {
     return addrs;
 }
 
-function drawMainMap(addrList) {
-    let map = L.map('main-map', {
-        center: [44.4781994, -73.2126357],
-        zoom: 20
-    });
+async function drawMainMap(addrList) {
+    let map = L.map('main-map').setView([44.4781994, -73.2126357],15);
```

Comment:
this is just another way to set up the map view

Reply:
Just need to get the markers to show up now.